business/src/main/java/org/mskcc/cbio/portal/model/Mutation.java

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (c) 2016 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
+ * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package org.mskcc.cbio.portal.model;
+
+import org.cbioportal.model.Gene;
+import org.cbioportal.model.GeneticProfile;
+import org.cbioportal.model.Sample;
+
+import java.io.Serializable;
+
+public class Mutation implements Serializable {
```

Comment:
@ecerami it's not, it's the one that has mutation event and all the other properties, I didn't delete it because if you remember in the first 2 sprints of first rearchitecturing effort with Ben and Rob, we changed mutation related `Dao`s to use MyBatis, so this model is used by those `Dao`s, so I moved it to a legacy module. @onursumer since everything in `business` module is legacy, I didn't change the name, all those name changes adding `Legacy` to the class name is because MyBatis throws error if there are multiple mappers in the system with the same name, but this class doesn't have that problem.